src/Avalonia.Base/Media/BackgroundSizing.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Avalonia.Media
+{
+    /// <summary>
+    /// Defines how a background is drawn relative to its border.
+    /// </summary>
+    public enum BackgroundSizing
```

Comment:
I debated if this should be in Avalonia.Media (base) or Avalonia.Controls. Finally it was decided to go in the lowest level that made sense.

 - It is useful in the new GeometryBuilder which should go with the other Geometry types in Avalonia.Media
 - WinUI upstream has this defined in DirectUI even lower than the UI framework: https://github.com/microsoft/microsoft-ui-xaml/blob/93742a178db8f625ba9299f62c21f656e0b195ad/dxaml/xcp/core/core/elements/framework.cpp#L3719C39-L3719C47